tests/test_dataset.py

```diff
@@ -192,18 +191,18 @@ def test_version_param(self, tmp_path):
         assert str(e.value) == 'The version parameter must be supplied a str.'
 
         name, version = 'gmb', ''
-        with pytest.raises(ValueError) as e:
+        with pytest.raises(KeyError) as e:
             load_dataset('gmb', version=version)
-        assert str(e.value) == (f'Failed to load dataset because "{version}" is not a valid PyDAX version for the '
-                                f'dataset "{name}". You can view all valid datasets and their versions by running the '
-                                'function pydax.list_all_datasets().')
+        assert str(e.value) == (f'\'"{version}" is not a valid PyDAX version for the dataset "{name}". '
```

Comment:
TIL: https://stackoverflow.com/a/24999035/1150462

Reply:
ohh that's why that is, I didn't see that comment when I google searched, thanks for linking. I only added the quotes in because pytest was complaining 😆